packages/utils/src/hooks/useWindowSize.ts

```diff
@@ -3,23 +3,24 @@ import { debounce } from '../helpers';
 
 export const useWindowSize = () => {
     const [windowSize, setWindowSize] = useState({
-        width: window.innerWidth,
-        height: window.innerHeight
+        width: typeof window !== 'undefined' && window.innerWidth,
```

Comment:
You should write it like this -> `typeof window !== 'undefined' ? window.innerWidth : 0`